app/scripts/app.mediumPosts.ts

```diff
@@ -0,0 +1,78 @@
+// Formating posts date
```

Comment:
File uses tabs. Default in project - spaces. Please enable editorconfig plugin in sublime.

Reply:
done